tools/osbuilder/rootfs-builder/cbl-mariner/Dockerfile.in

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2022 Dallas Delaney, 2022 Microsoft Inc.
```

Comment:
Should it be 2023?

Reply:
Thanks for catching this, it will be changed to 2023. I'm also wondering if there's guidance for including an author name in the copyright. It looks like Ubuntu follows this pattern but the rest of the distros under `rootfs-builder` do not.

Reply:
Typically I'm not used to seeing the author's name in copyright statements, just the company name, but maybe that's just as mostly we work for companies that own the IP we produce, it might be that in the ubuntu case that an individual contributor not working for a corporation worked on it. That said I don't think there is a contribution standard in the community that mandates it (if anyone can correct that would be helpful for me too) and my experience of previous open-source projects was that different companies' lawyers had different requirements, so there was variations.

Reply:
It looks like the copyright for Ubuntu is indeed what you suggested. I've gone ahead and updated the PR to only include the company name and correct year.